compute/resource_job_test.go

```diff
@@ -452,3 +385,157 @@ func TestResourceJobDelete_Error(t *testing.T) {
 	qa.AssertErrorStartsWith(t, err, "Internal error happened")
 	assert.Equal(t, "789", d.Id())
 }
+
+func TestResourceJobCreateSingleNode(t *testing.T) {
+	cluster := Cluster{
+		NumWorkers: 0, SparkVersion: "7.3.x-scala2.12", NodeTypeID: "Standard_DS3_v2",
+		SparkConf: map[string]string{
+			"spark.master":                     "local[*]",
+			"spark.databricks.cluster.profile": "singleNode",
+		},
+	}
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "POST",
+				Resource: "/api/2.0/jobs/create",
+				ExpectedRequest: JobSettings{
+					NewCluster: &cluster,
+					SparkJarTask: &SparkJarTask{
+						MainClassName: "com.labs.BarMain",
+					},
+					Libraries: []Library{
```

Comment:
please remove `libraries` block, it's tested in previous cases, let's focus these corner case tests on more narrow scope and fill in only required params